core/src/main/groovy/noe/rhel/server/workspace/WorkspaceHttpdTomcatsBaseOS.groovy

```diff
@@ -18,7 +19,8 @@ class WorkspaceHttpdTomcatsBaseOS extends WorkspaceMultipleTomcats{
 
   protected static WorkspaceHttpdBaseOS workspaceHttpd
   protected static WorkspaceTomcat workspaceTomcat
-  private List<File> modJkModules = []
+  private List<File> moduleFiles = []
+  private FileStateVault fileStateVault = new FileStateVault()
```

Comment:
This is never used

Reply:
Change in execution code made, present in next commit